KISS crashes with disabled work profile

**To Reproduce**
Steps to reproduce the crash:
1. Install Shelter, set device PIN and use the same PIN with work profile
2. Clone one app to shelter
3. Reset KISS history (or not)
4. Try to search something with disabled work profile, instant crash

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Stack trace**
12-13 19:12:19.238  6909  6909 D AndroidRuntime: Shutting down VM
12-13 19:12:19.238  6909  6909 E AndroidRuntime: FATAL EXCEPTION: main
12-13 19:12:19.238  6909  6909 E AndroidRuntime: Process: fr.neamar.kiss, PID: 6909
12-13 19:12:19.238  6909  6909 E AndroidRuntime: java.lang.IllegalStateException: User 14 is locked or not running
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Parcel.createException(Parcel.java:2079)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:2039)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Parcel.readException(Parcel.java:1987)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.content.pm.ILauncherApps$Stub$Proxy.getShortcuts(ILauncherApps.java:1271)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.content.pm.LauncherApps.getShortcuts(LauncherApps.java:917)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at fr.neamar.kiss.result.ShortcutsResult.getDrawable(ShortcutsResult.java:12)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at fr.neamar.kiss.result.ShortcutsResult.display(ShortcutsResult.java:33)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at fr.neamar.kiss.adapter.RecordAdapter.getView(RecordAdapter.java:1)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.AbsListView.obtainView(AbsListView.java:2405)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.ListView.makeAndAddView(ListView.java:2067)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.ListView.fillUp(ListView.java:828)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.ListView.layoutChildren(ListView.java:1805)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.AbsListView.onLayout(AbsListView.java:2202)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onLayout(DecorView.java:779)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.View.layout(View.java:22466)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6304)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3157)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2614)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1745)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7768)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:967)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:791)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:726)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:883)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: Caused by: android.os.RemoteException: Remote stack trace:
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.server.pm.ShortcutService.throwIfUserLockedL(ShortcutService.java:1193)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.server.pm.ShortcutService$LocalService.getShortcuts(ShortcutService.java:2582)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at com.android.server.pm.LauncherAppsService$LauncherAppsImpl.getShortcuts(LauncherAppsService.java:670)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.content.pm.ILauncherApps$Stub.onTransact(ILauncherApps.java:589)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 	at android.os.Binder.execTransactInternal(Binder.java:1021)
12-13 19:12:19.238  6909  6909 E AndroidRuntime: 


**Additional information**
 - App version: 3.15.0
 - App installed via: (f-droid, Play Store, debug APK): f-droid

- LineageOS 17.1
- Shelter 

**Additional context**

https://github.com/Neamar/KISS/blob/59057a489388fdd913e28293c5d4ce99d2fda170/app/src/main/java/fr/neamar/kiss/result/ShortcutsResult.java#L116

https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/content/pm/LauncherApps.java#1054

--- a/app/src/main/java/fr/neamar/kiss/result/ShortcutsResult.java
+++ b/app/src/main/java/fr/neamar/kiss/result/ShortcutsResult.java
@@ -144,7 +144,6 @@ public class ShortcutsResult extends Result {
         Drawable shortcutDrawable = null;
         if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
             final LauncherApps launcherApps = (LauncherApps) context.getSystemService(Context.LAUNCHER_APPS_SERVICE);
-            UserManager userManager = (UserManager) context.getSystemService(Context.USER_SERVICE);
             assert launcherApps != null;
 
             if (launcherApps.hasShortcutHostPermission() && !TextUtils.isEmpty(shortcutPojo.packageName)) {
@@ -157,11 +156,13 @@ public class ShortcutsResult extends Result {
 
                 // Find the correct UserHandle, and retrieve the icon.
                 for (UserHandle userHandle : userHandles) {
-                    if (userManager.isUserRunning(userHandle)) {
+                    try {
                         List<ShortcutInfo> shortcuts = launcherApps.getShortcuts(query, userHandle);
                         if (shortcuts != null && shortcuts.size() > 0) {
                             shortcutDrawable = launcherApps.getShortcutIconDrawable(shortcuts.get(0), 0);
                         }
+                    } catch (IllegalStateException ignored) {
+                        // do nothing if user is locked or not running
                     }
                 }
             }

